ui/src/components/pages/TaskListScreen.js

```diff
@@ -28,6 +28,10 @@ const TasksScreen = (props) => {
     fetchUsers();
   }, [sendRequest, params.userId]);
 
+  handleDelete = () => {
+    alert("Task deleted");
+  };
```

Comment:
you are on the right track and would just need the task's ID in this case, then your http request outwards
```suggestion
  onDeleteHandler = (taskId) => {
    alert(`Task deleted: ${taskId}`);
    // post request out to server goes here
  };
```